lit-base/model_service.py

```diff
@@ -0,0 +1,91 @@
+import mxnet as mx
+import tempfile
+import urllib2
+import numpy as np
+from PIL import Image
+
+from model_loader import sym, arg_params, aux_params
+
+from collections import namedtuple
+from StringIO import StringIO
+
+
```

Comment:
Do you plan to retain this code when you support MAR? If not lets skip reviewing this.